ChangeLog.md

```diff
@@ -0,0 +1,23 @@
+# [if me](https://www.if-me.org/) ChangeLog
+
+## Major bug fixes and noteworthy changes
+
+*2017-09-27
```

Comment:
Are you trying to format this as a bullet? If so, you need to put a space between the bullet and the text.

Reply:
Oh, thanks for looking at this. No, not a bullet; what I wanted here was just something to make the _date_ stand out a little.

tbh, I'm not great with layout and formatting. What I'll do is post the raw markdown in #design and ask for feedback from the pros. Let's not merge this yet.

Reply:
This is a helpful Markdown [cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). Try using headings :)